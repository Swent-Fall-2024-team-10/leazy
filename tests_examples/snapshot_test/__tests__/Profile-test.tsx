import React from 'react'
import { render } from '@testing-library/react-native'
import Profile from '../ExampleSnapshot'

describe('Profile snapshots', () => {
    
    it('renders correctly', () => {
        const { toJSON } = render(<Profile name="Adrien Deschenaux" activity="EPFL Student" />);
        expect(toJSON()).toMatchSnapshot();
    });

    it('renders correctly', () => {
        const { toJSON } = render(<Profile name="Voltaire" activity="Philosopher" />);
        expect(toJSON()).toMatchSnapshot();
    });

    it('renders correctly', () => {
        const { toJSON } = render(<Profile name="Claude Debussy" activity="Composer" />);
        expect(toJSON()).toMatchSnapshot();
    });
});