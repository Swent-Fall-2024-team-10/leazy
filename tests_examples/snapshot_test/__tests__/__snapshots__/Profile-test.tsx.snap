// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile snapshots renders correctly 1`] = `
<View>
  <Text>
    Adrien Deschenaux
  </Text>
  <Text>
    EPFL Student
  </Text>
</View>
`;

exports[`Profile snapshots renders correctly 2`] = `
<View>
  <Text>
    Voltaire
  </Text>
  <Text>
    Philosopher
  </Text>
</View>
`;

exports[`Profile snapshots renders correctly 3`] = `
<View>
  <Text>
    Claude Debussy
  </Text>
  <Text>
    Composer
  </Text>
</View>
`;
